integrations/atlassian/jira/oauth.go

```diff
@@ -46,15 +48,15 @@ func (h handler) handleOAuth(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	url, err := apiBaseURL()
+	u, err := APIBaseURL()
```

Comment:
I sometimes go the other way to avoid name collision:

```go
import urllib "net/url"
// ...
url, err := urllib.JoinPath(url, "/ex/jira", res[0].ID)
```
```